canonical-websites/www.ubuntu.com
# Add appstore whitepaper landing page

## Done

* Add appstore whitepaper landing page
* [Copydoc](https://docs.google.com/document/d/1PM7HoXgECZQSPDsMOM9h-4YsEvHhiilUV5iZovqFuxY)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/engage/iot-business-model](http://0.0.0.0:8001/engage/iot-business-model)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

## Notes

* Form ID + success page changes needed before merging
